manifests/charts/istio-control/istio-discovery/files/injection-template.yaml

```diff
@@ -343,8 +343,14 @@ spec:
       runAsGroup: 1337
       fsGroup: 1337
       {{ if or (eq (annotation .ObjectMeta `sidecar.istio.io/interceptionMode` .ProxyConfig.InterceptionMode) `TPROXY`) (eq (annotation .ObjectMeta `sidecar.istio.io/capNetBindService` .Values.global.proxy.capNetBindService) `true`) -}}
+      {{ if eq (annotation .ObjectMeta `sidecar.istio.io/interceptionMode` .ProxyConfig.InterceptionMode) `TPROXY` -}}
+      runAsNonRoot: false
+      runAsUser: 1337
+      {{- end }}
+      {{ if eq (annotation .ObjectMeta `sidecar.istio.io/capNetBindService` .Values.global.proxy.capNetBindService) `true` -}}
       runAsNonRoot: false
       runAsUser: 0
+      {{- end }}
```

Comment:
These 2 `if` are not exclusive. IT means runAsNonRoot and runAsUser could appear twice.

Forgive my lack of knowledge on template language: what is the corresponding `if` of the `else` on line 354?

Reply:
Ok, the else matches the `if` w/o `end`

Reply:
Ah. Will fix.